user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_behavioral_anomaly_detection.md

```diff
@@ -8,6 +8,7 @@ uid: Working_with_behavioral_anomaly_detection
 >
 > - This feature requires [Storage as a Service](xref:STaaS) or a [self-hosted Cassandra-compatible database](xref:Supported_system_data_storage_architectures).
 > - Anomaly detection is only available for numeric parameters that are not part of [partial tables](xref:Table_parameters#partial-tables). It is also limited to at most 100,000 parameters per DMA.
+> - From DataMiner 10.4.2/10.5.0 onwards<!--RN 38041-->, trend data is limited to at most 250,000 trended parameters per DMA.
```

Comment:
The current phrasing could be misinterpreted as applying to trend data in general, rather than to the calculation of state icons by DataMiner Analytics. Perhaps you could instead add this info in the notes under "Trend icons on data pages" (e.g. on line 74 below), and rephrase it to e.g. "As a safeguard, from DataMiner 10.4.2/10.5.0 onwards<!--RN 38041-->, the number of parameters for which state icons are calculated is limited to at most 250,000 trended parameters per DMA."